interface/super/manage_site_files.php

```diff
@@ -209,6 +259,28 @@ function msfFileChanged() {
 </center>
 
 </form>
+
+<div id="generate_thumb">
+    <table>
+        <tr>
+            <td class="thumb_title" width="19%">
+                <b><?php echo xl('Generate Thumbnails')?></b>
+            </td>
+            <td class="thumb_subtitle" width="28%">
+                <span><?php echo xl('The thumbnails will be store ' . $storage_method) ?></span>
```

Comment:
Do note that each thumbnail needs to use whatever method the document uses for storage of the thumbnail. It does not matter what the globals is set to. For example, a installation may be using both hard drive and couchdb (if it changed from one to the other). So, when your are creating the thumbnails, you will use the setting for each document when you decide whether to use hard drive or couchdb.